xbmc/utils/FileUtils.cpp

```diff
@@ -162,3 +163,72 @@ bool CFileUtils::RemoteAccessAllowed(const std::string &strPath)
   }
   return false;
 }
+
+std::string CFileUtils::GetDateAdded(const std::string& strFileNameAndPath, const int& iAdvancedSettingsDateAdded)
+{
+  if (strFileNameAndPath.empty())
+    CLog::Log(LOGDEBUG, "%s empty strFileNameAndPath variable", __FUNCTION__);
+  return "";
```

Comment:
wohhhh .. you forgot to enclose the return? this will return empty for every call to this method